docs/src/clusters.md

```diff
@@ -97,8 +97,7 @@ $ solana-validator \
     --identity validator-keypair.json \
     --vote-account vote-account-keypair.json \
     --known-validator 5D1fNXzvv5NjV1ysLjirC4WY92RNsVH18vjmcszZd8on \
-    --known-validator 7XSY3MrYnK8vq693Rju17bbPkCN3Z7KvvfvJx4kdrsSY \
-    --known-validator Ft5fbkqNa76vnsjYNwjDZUXoTWpP7VYm3mtsaQckQADN \
```

Comment:
@leoluk y'all planning on bringing this guy back on line?

Reply:
Yes, will be back online shortly on nicer hardware

Reply:
Oh, Good to know. I'll leave you on the list